tests/step-110.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+# Test:
+#   Trigger hooks on a bare repo with a push from a local repo.
+
+if ! curl --version && ! wget --version; then
```

Comment:
Why does this test need `curl` or `wget`?

Reply:
Oh I dont know... it does a update I think, can I remove this, need to look what to do for the install...

Reply:
`curl` or `wget` should only be needed if the test explicitly tests downloading something, otherwise it shouldn't be skipped just because these are missing.